static/js/server_events_dispatch.js

```diff
@@ -758,6 +760,9 @@ export function dispatch_normal_event(event) {
                     }
                     break;
                 case "remove":
+                    if (hash_util.is_editing_group(event.group_id)) {
+                        user_group_edit.open_right_panel_empty();
+                    }
```

Comment:
I think this block would be more nicely written as a `user_group_edit.handle_deleted_group(event.group_id)` -- i.e. adding a small function in the other file that handles the event fully for the purpose of the `user_group_edit` module.

Reply:
Made the changes as suggested.